src/plugins/discover/public/__mocks__/services.ts

```diff
@@ -236,6 +242,11 @@ export function createDiscoverServicesMock(): DiscoverServices {
     contextLocator: { getRedirectUrl: jest.fn(() => '') },
     singleDocLocator: { getRedirectUrl: jest.fn(() => '') },
     urlTracker: urlTrackerMock,
+    profilesManager: new ProfilesManager(
```

Comment:
**doubt**: should this be the same as the mock in https://github.com/elastic/kibana/pull/183797/files#diff-66364df80226d194a5368e2cc7208a0f29538806fb2865d46a82e8f97e1bf3ddR86 ?

Reply:
It should be! Updated here: https://github.com/elastic/kibana/pull/183797/commits/f5fd8429230ff03a347f48ff5396644eb28e8d69.